pillar.example

```diff
@@ -1,8 +1,8 @@
 collectd:
   FQDNLookup: true
   TypesDB: ['/usr/share/collectd/types.db']
+  default_plugins: [battery, cpu, entropy, load, memory, swap, users]
```

Comment:
I don't disagree on your concept, but the pillar data in the example as it currently is in master is incorrect. I did not correct the underlying data model discrepancy - as that would be a backwards-incompatible modification - I simply corrected the documentation to reflect what actually is expected.